registries/docker-daemon/start.sh

```diff
@@ -5,7 +5,10 @@ echo "Installing a daemon registry"
 docker run -d -p 5000:5000 --name registry registry:2
 
 registry_ip=$(docker inspect --format "{{.NetworkSettings.IPAddress }}" registry)
-sudo su -c "echo \"${registry_ip} registry.kube-system.svc.cluster.local\" >> /etc/hosts"
+if ! grep registry /etc/hosts; then
```

Comment:
how about we remove any entry from hosts with `registry.kube-system.svc.cluster` and then add the new value?

Reply:
You can't edit /etc/hosts in a container. This seemed like the least worst compromise. I guess we could attempt to edit it and fall back to this if it fails.

Reply:
I'm open to finding a better way to define a one-off dns record. Until then, we can make the test more robust. The run will fail if the IP is not correct.

```suggestion
if ! grep registry.kube-system.svc.cluster.local /etc/hosts; then
```